tests/test_samples.py

```diff
@@ -1339,3 +1339,38 @@ def test_old_gui():
         make_2d_axes(['x0', 'y0'], tex={'x0': '$x_0$', 'y0': '$y_0$'})
     with pytest.raises(NotImplementedError):
         make_1d_axes(['x0', 'y0'], tex={'x0': '$x_0$', 'y0': '$y_0$'})
+
+
+def test_groupby_stats():
+    mcmc = read_chains('./tests/example_data/cb')
+    chains = mcmc.groupby('chain')
```

Comment:
> Getting performance warnings /Users/adam/programming/anesthetic/anesthetic/weighted_pandas.py:27: PerformanceWarning: dropping on a non-lexsorted multi-index without a level parameter may impact performance.

This above line is the culprit, which got changed in https://github.com/williamjameshandley/anesthetic/pull/272/commits/c49536210ac8b3c74f99b548802d73737e04bb1a from:
```python
chains = mcmc.groupby(('chain', '$n_\\mathrm{chain}$'), group_keys=False)
```
Since we have a MultiIndex, we will get the warning when we only use the column name `'chain'` without its label. How do we prefer to handle this? It is good to know that things will work even when calling with only `'chain'`. But it's also nice to keep our test feedback clean. Shall we keep things as is and turn off the check for a PerformanceWarning for this line? Or revert to grouping with a joint call of column name and label?

Reply:
I'd prefer a joint call of column name and label, as warning suppression is quite messy

Reply:
Happy to go with @Ormorod's suggestion.